src/components/entity/state-info.ts

```diff
@@ -120,7 +120,8 @@ class StateInfo extends LitElement {
       .name {
         @apply --paper-font-common-nowrap;
```

Comment:
I don't think this works, can we check what was in here and see what else is missing?

Reply:
(Same for line 97) cc: @iantrich

Reply:
That is what was there, but I think you're right in that `@apply‘ does not work with Lit. That's a Polymer thing, I believe. So I totally missed that part of the conversion.